docs/Streams.md

```diff
@@ -186,3 +194,21 @@ MsQuic does not enforce it, and it is legal for an application to provide less b
 
 After the initial receive window is full, flow control will ensure that the peer does not send more data than there is buffer space available.
 However, the application should still provide enough buffer space to keep flow control from impacting performances.
+
+## Receive Shutdown
+
+The receiver can abortively shutdown a stream receive direction by calling [`StreamShutdown`](api/StreamShutdown.md) 
+with the `QUIC_STREAM_SHUTDOWN_FLAG_ABORT_RECEIVE` option.
+
+# Closing a Stream
+
+Once all the directions relevant for a stream have been shutdown, the application receives a
+`QUIC_STREAM_EVENT_SHUTDOWN_COMPLETE` event.
+
+The application must then close the stream using [`StreamClose`](api/StreamClose.md), and can release its notification
+handling context once the call returns.
+
+If the app closes a stream before it is shutdown, the stream will be shutdown abortively with an error code of `0`.
+This should be avoided, the app should rather abortively shutdown the stream first with a meaningful error code.
```

Comment:
I think this would read better as
> This should be avoided; instead, the app should abortively shutdown the stream first with a meaningful error code.